minigunapp/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
No real reason other than that it's a quick prototype. I could write some if you like. I had never used django so if I'm in a space where I'm not sure what's possible or what it would look like yet, I'm not usually writing tests first. Like I wasn't even sure if mongo could actually be made to work out with this so I didn't write tests around checking the database to see if things were correct.